usagov/usagov-2021
# usagov-993-taxonomy-datalayer-array: add logic for states and agencie…

…s datalayer


## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-993

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Test Agencies Page Datalayer
- Goto an agency record http://localhost/agencies/defense-finance-and-accounting-service-debt-and-claims-management-center and "View Page Source"
-You should see the datalayer reflect the agencies page breadcrumb
"Taxonomy_Text_1": "Home",
 "Taxonomy_Text_2": "About the U.S. and its government",
 "Taxonomy_Text_3": "A-Z index of U.S. government departments and agencies",
 "Taxonomy_Text_4": "Defense Finance and Accounting Service Debt and Claims Management Center",
  "Taxonomy_Text_5": "Defense Finance and Accounting Service Debt and Claims Management Center",
  "Taxonomy_Text_6": "Defense Finance and Accounting Service Debt and Claims Management Center"
  "Taxonomy_Text_1": "/",
  "Taxonomy_Text_2": "/about-the-us",
  "Taxonomy_Text_3": "/agency-index",
  "Taxonomy_Text_4": "/agencies/defense-finance-and-accounting-service-debt-and-claims-management-center",
  "Taxonomy_Text_5": "/agencies/defense-finance-and-accounting-service-debt-and-claims-management-center",
  "Taxonomy_Text_6": "/agencies/defense-finance-and-accounting-service-debt-and-claims-management-center"


- Test States Page Datalayer
- Goto a state page http://localhost/states/american-samoa
- You should see the datalayer reflect the states page breadcrumb
"Taxonomy_Text_1": "Home",
 "Taxonomy_Text_2": "About the U.S. and its government",
  "Taxonomy_Text_3": "State governments",
  "Taxonomy_Text_4": "American Samoa",
  "Taxonomy_Text_5": "American Samoa",
  "Taxonomy_Text_6": "American Samoa"
  "Taxonomy_Text_1": "/",
  "Taxonomy_Text_2": "/about-the-us",
  "Taxonomy_Text_3": "/state-governments",
  "Taxonomy_Text_4": "/states/american-samoa",
   "Taxonomy_Text_5": "/states/american-samoa",
   "Taxonomy_Text_6": "/states/american-samoa"

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
